### ISSUE TITLE
Labeling through google drive cannot be confirmed when connected to Tor

### ISSUE THREAD

**Issue by STew790** [trezor/trezor-suite#11144]
**Describe the bug**
When I tried to connect labels via google drive, a window to confirm is too wide and cannot be confirmed because the buttons are out of screen. (screenshot attached)

**Steps to reproduce:**
1. Go to settings
2. Turn ON Tor
3. Turn ON labeling
4. Try to connect to Google Drive

**Info:**
 - Suite version: desktop 24.2.1 (fd5d1fa8475b78659e80f19bada6933ca804fab3)
 - Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/24.2.1 Chrome/118.0.5993.129 Electron/27.0.4 Safari/537.36
 - OS: Linux x86_64
 - Screen: 1920x1080
 - Device: Trezor T2T1 2.6.4 regular (revision 42e9ed0e09033d474dee1a560fe5870646fa440e)
 - Transport: BridgeTransport 2.0.32

**Expected result:**
I can confirm connecting to Google Drive in the confirmation window

**Actual result:**
Confirmation window for connecting to Google Drive is too wide and buttons aren't visible.

**Screenshots:**
![image](https://github.com/trezor/trezor-suite/assets/48477147/4992f1c0-dba6-43db-adc3-daf84d629421)

**Comment by mroz22** [comment]
First of all it should not open electron window but native browser instead.

**Comment by STew790** [comment]
Yes, I was just trying to reproduce this on production and it just opens up the browser immediately, so I tried the 24.2.1 version again and it is not showing me the electron window now and goes straight into browser.
I will try to reproduce this.

**Comment by STew790** [comment]
Ok sorry, I just realised what was the case. I had TOR on and Suite asks me every time when I am to be redirected to outside of Suite if I really want to go to that URL. And in combination with Google drive, you get this. I updated the reproduction steps.

**Comment by jvaclavik** [comment]
Probably not relevant anymore with sync, right @hynek-jina ?

**Comment by hynek-jina** [comment]
Legacy labeling is still available.. so the fix would be still useful.. Just the priority is lowered...
